packages/shared/src/components/modals/post/boost/AdsDashboardModal.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import { Modal } from '../../common/Modal';
+import type { ModalProps } from '../../common/Modal';
+import { Typography, TypographyType } from '../../../typography/Typography';
+import { CoreIcon } from '../../../icons';
+import { IconSize } from '../../../Icon';
+import type { PostCampaign } from '../../../../hooks/post/usePostBoost';
+import { usePostBoost } from '../../../../hooks/post/usePostBoost';
+import { DataTile } from '../../../../features/boost/DataTile';
+import { BoostHistoryLoading } from '../../../../features/boost/BoostHistoryLoading';
+import { CampaignList } from '../../../../features/boost/CampaignList';
+
+export function AdsDashboardModal(props: ModalProps): ReactElement {
+  const { data, isLoading } = usePostBoost();
+  const [campaign, setCampaign] = React.useState<PostCampaign>(null);
+
+  return (
+    <Modal
+      {...props}
+      isOpen
+      kind={Modal.Kind.FixedCenter}
+      size={Modal.Size.Small}
+    >
+      <Modal.Header>
```

Comment:
Isn't it enough to pass header or titlte here?
Think it takes uniform classes then